public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```

Comment:
Possibly want a better description

Reply:
These are defaults that are set when setting up a React app. @HamadAli248 should have changed these in his branch - I will check. This commit was only meant to get the template on to master for Hamad to continue working on it.